.github/workflows/codeql.yml

```diff
@@ -0,0 +1,93 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "trunk" ]
+  pull_request:
+    branches: [ "*" ]
+  schedule:
+    - cron: '33 23 * * 4'
+
+jobs:
+  analyze:
+    name: Analyze (${{ matrix.language }})
+    # Runner size impacts CodeQL analysis time. To learn more, please see:
+    #   - https://gh.io/recommended-hardware-resources-for-running-codeql
+    #   - https://gh.io/supported-runners-and-hardware-resources
+    #   - https://gh.io/using-larger-runners (GitHub.com only)
+    # Consider using larger runners or machines with greater resources for possible analysis time improvements.
+    runs-on: ${{ (matrix.language == 'swift' && 'macos-latest') || 'ubuntu-latest' }}
+    timeout-minutes: ${{ (matrix.language == 'swift' && 120) || 360 }}
+    permissions:
+      # required for all workflows
+      security-events: write
+
+      # required to fetch internal or private CodeQL packs
+      packages: read
+
+      # only required for workflows in private repositories
+      actions: read
+      contents: read
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - language: java-kotlin
+          build-mode: manual
+        - language: ruby
+          build-mode: none
+        - language: swift
+          build-mode: manual
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - if: matrix.language == 'java-kotlin'
+      uses: actions/setup-java@v4
+      with:
+        java-version: '17'
+        distribution: corretto
+
+    - name: Install NDK
+      if: matrix.language == 'java-kotlin'
+      uses: nttld/setup-ndk@v1
+      with:
+        ndk-version: "r25b"
+        link-to-sdk: true
+
+    # Initializes the CodeQL tools for scanning.
+    - name: Initialize CodeQL
+      uses: github/codeql-action/init@v3
+      with:
+        languages: ${{ matrix.language }}
+        build-mode: ${{ matrix.build-mode }}
+        # If you wish to specify custom queries, you can do so here or in a config file.
+        # By default, queries listed here will override any specified in a config file.
+        # Prefix the list here with "+" to use these queries and those in the config file.
+
+        # For more details on CodeQL's query packs, refer to: https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning#using-queries-in-ql-packs
+        # queries: security-extended,security-and-quality
+
+    - name: Restore Rust Cache
+      uses: Swatinem/rust-cache@v2
+      with:
+        key: "${{ matrix.language }}"
+
+    - name: Prepare Kotlin Project
+      if: matrix.language == 'java-kotlin'
+      shell: bash
+      run: |
+        make setup-rust-android-targets
+        cd native/kotlin
+        ./gradlew cargoBuild --info
```

Comment:
What's the intention here? `./gradlew cargoBuild` is basically the same as running `cargo build --release -p wp_api`. (not exactly, but very minor differences) So, it doesn't really build anything Kotlin related.

You probably want something like `./gradlew :api:android assemble` instead, but hard to say without knowing the intention.

Reply:
You're right, but I'm getting this error:

> Cannot locate tasks that match ':api:android' as task 'android' not found in project ':api'.

Am I holding it wrong?

Reply:
Oh, sorry, it looks like I put a space where I should have put a `:` 🤦 It should be `./gradlew :api:android:assemble`. Sorry about that!

Having said that, maybe do `./gradlew :api:android:assembleRelease` instead. We don't have any debug/release specific config, and running `assemble` would do both. So, `assembleRelease` should be faster without any downside.